src/main/java/htsjdk/samtools/cram/compression/rans/Constants.java

```diff
@@ -1,7 +1,9 @@
 package htsjdk.samtools.cram.compression.rans;
 
-final class Constants {
-    static final int TF_SHIFT = 12;
-    static final int TOTFREQ = (1 << TF_SHIFT); // 4096
-    static final int RANS_BYTE_L = 1 << 23;
-}
+final public class Constants {
+    public static final int TF_SHIFT = 12;
+    public static final int TOTFREQ = (1 << TF_SHIFT); // 4096
+    public static final int RANS_BYTE_L_4x8 = 1 << 23;
+    public static final int RANS_BYTE_L_Nx16 = 1 << 15;
```

Comment:
`RANS_BYTE_L...` represents the lower bound for renormalization ? Could use a better name.